it/uk/gov/hmrc/agentsubscriptionfrontend/support/EndpointBehaviours.scala

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.support
+
+import play.api.mvc.{AnyContentAsEmpty, Result}
+import play.api.test.FakeRequest
+import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
+import uk.gov.hmrc.agentsubscriptionfrontend.stubs.AuthStub
+import uk.gov.hmrc.agentsubscriptionfrontend.support.SampleUsers.individual
+import uk.gov.hmrc.play.test.UnitSpec
+
+trait EndpointBehaviours {
+  me: UnitSpec with WireMockSupport =>
+
+  protected def anAgentAffinityGroupOnlyEndpoint(doRequest: FakeRequest[AnyContentAsEmpty.type] => Result) = {
```

Comment:
IntelliJ wants this to have a return type specified because it is a protected (not private) method, would be nice to add this (`: Unit`) just to keep this file free of warnings.